docs/tutorials/manifest-parser.md

```diff
@@ -346,8 +346,8 @@ MyManifestParser.prototype.loadReference_ =
 
 If your content is encrypted, there are a few changes to the manifest you need
 to do.  First, for each Stream that contains encrypted content, you need to set
-`stream.encrypted` to true and put the key IDs that the stream is encrypted with
-in the `stream.keyIds` set.  Filling out the `keyIds` is technically optional,
+`stream.encrypted` to true and put the key IDs that the stream is encrypted with in
+the `stream.keyIds` set.  Filling out the `keyIds` is technically optional,
```

Comment:
Not sure why you moved the word "in" from one line to another.  Am I overlooking some other change in the diff?

Reply:
Originally, it was "with in", but to me, it seemed like using "within" would be the right approach. Later, @theodab suggested the changes, so I reverted it back to the original state. That's just what I did.

Reply:
I think as programmers, we could disambiguate things if English were parenthetical.  For example, this is how you should understand the sentence in your head:

> You need to ((set `stream.encrypted` to true) and (put (the key IDs that the stream is encrypted with) in (the `stream.keyIds` set)).

There is an obnoxious rule in English that you should not end sentences with a preposition.  In practice, of course, people do this all the time.  Here, we have a phrase that ends with a preposition: "the key IDs that the stream is encrypted with".  It could be rephrased as "the key IDs with which the stream is encrypted", if you think that would help with clarity from your perspective.

Another small change that could clarify is that the word "in" could become "into" (as in "into the keyIds set").  If you think that would also help with clarity, I think that would be a reasonable improvement.

We recognize that with developers contributing from all over the world, it is really helpful to have docs that are understandable to non-native English speakers.  So we are definitely motivated by arguments about clarity.  If you didn't understand what we meant here, it should be fixed!  After that, it's just a matter of finding the right fix which makes sense to everyone.

Thanks!